openapi/openapi.yaml

```diff
@@ -607,6 +607,13 @@ paths:
           schema:
             type: string
           description: URL of the GitHub repository containing the Pull Request.
+        - name: kebechet_metadata
+          in: query
+          required: false
+          schema:
+            type: object
+            additionalProperties: true
```

Comment:
It might be good to be more strict here as this is the entrypoint to the system. IIRC you wanted to validate this also in the messaging?

Reply:
Yeah, I can be more specific here. Once we are using pydantic then messaging will have built in validation so it shouldn't matter as much

Reply:
sounds good :+1: